docs/ci-integration/guides/bitbucket-pipelines-integration.md

```diff
@@ -0,0 +1,40 @@
+# Bitbucket Pipelines integration
+
+Bitbucket Pipelines run in a shared Docker environment and do not support running Earthly builds directly due to [restrictions](https://jira.atlassian.com/browse/BCLOUD-21419) that Bitbucket has put in place.
+
+You can however, run Earthly builds on Bitbucket pipelines via [Earthly Satellites](../../cloud/satellites.md). Because Bitbucket Pipelines run as containers you can also use the official Earthly Docker image. Here is an example of a Bitbucket Pipeline build. This example assumes `+unit-test` and `+build` targets in your Earthfile.
+
+```yml
+# ./bitbucket-pipelines.yml
+
+image: earthly/earthly:latest
+
+pipelines:
+  default:
+    - step:
+        name: "Set Earthly token"
+        script:
+          - export EARTHLY_TOKEN=$EARTHLY_TOKEN
+          # See https://docs.earthly.dev/docs/earthly-command#earthly-account-create-token to obtain a token.
+          - earthly --version
+    - step:
+        name: "Docker login"
+        script:
+          - docker login --username "$DOCKERHUB_USERNAME" --password "$DOCKERHUB_TOKEN"
+    - step:
+        name: "Unit tests"
+        script:
+          - earthly --sat $EARTHLY_SAT --org $EARTHLY_ORG +unit-test
+    - step:
+        name: "Build"
+        script:
+          - earthly --push --sat $EARTHLY_SAT --org $EARTHLY_ORG +build
```

Comment:
It's generally advisable to merge these in a single Earthly run, because then the Buildkit/Earthly magic can maximize parallelism.

Reply:
Done